### DISCUSSION TITLE
Issue with offline feedback

### DISCUSSION THREAD

**Discussion by cngirwa** [LiveHelperChat/livehelperchat#1949] [Q&A]
Hello Admin and all!

I hosted LHC and configured https also removed index.php by following steps suggested on lhc website. However, when enable offline messaging i get this error

>Warning: Missing boundary in multipart/form-data POST data in Unknown on line 0
{"success":true}

What seems to be the issue here?

Regards,

**Comment by remdex** [comment]
Sorry, but here all works :)
